nextflow_schema.json

```diff
@@ -438,197 +476,238 @@
                     "default": "GATK.GRCh38",
                     "fa_icon": "fas fa-book",
-                    "help_text": "If using a reference genome configured in the pipeline using iGenomes, use this parameter to give the ID for the reference. This is then used to build the full paths for all required reference genome files e.g. `--genome GRCh38`.\n\nSee the [nf-core website docs](https://nf-co.re/usage/reference_genomes) for more details."
-                },
-                "ascat_alleles": {
-                    "type": "string",
-                    "fa_icon": "fas fa-file",
-                    "description": "Path to ASCAT allele zip file. If you use AWS iGenomes, this has already been set for you appropriately."
-                },
-                "ascat_chromosomes": {
-                    "type": "string",
-                    "description": "Chromosomes to run ASCAT on.",
-                    "fa_icon": "fa-solid fa-text",
-                    "default": "c(1:22, 'X')",
-                    "help_text": "Specify specific chromosomes to run ASCAT on, i.e 'c('21', '22')'."
+                    "help_text": "If using a reference genome configured in the pipeline using iGenomes, use this parameter to give the ID for the reference. This is then used to build the full paths for all required reference genome files e.g. `--genome GRCh38`.\n\nSee the [nf-core website docs](https://nf-co.re/usage/reference_genomes) for more details.",
+                    "enum": [
+                        "GATK.GRCh38",
+                        "GATK.GRCh37",
+                        "Ensembl.GRCh37",
+                        "NCBI.GRCh38",
+                        "GRCm38",
+                        "TAIR10",
+                        "EB2",
+                        "UMD3.1",
+                        "WBcel235",
+                        "CanFam3.1",
+                        "GRCz10",
+                        "BDGP6",
+                        "EquCab2",
+                        "EB1",
+                        "Galgal4",
+                        "Gm01",
+                        "Mmul_1",
+                        "IRGSP-1.0",
+                        "CHIMP2.1.4",
+                        "Rnor_5.0",
+                        "Rnor_6.0",
+                        "R64-1-1",
+                        "EF2",
+                        "Sbi1",
+                        "Sscrofa10.2",
+                        "AGPv3",
+                        "hg38",
+                        "hg19",
+                        "mm10",
+                        "bosTau8",
+                        "ce10",
+                        "canFam3",
+                        "danRer10",
+                        "dm6",
+                        "equCab2",
+                        "galGal4",
+                        "panTro4",
+                        "rn6",
+                        "sacCer3",
+                        "susScr3"
+                    ]
```

Comment:
we can have `null` as well when using `--igenomes_ignore`

Reply:
🤦‍♀️ right, can I add it here or remove the whole list?

Reply:
We have checks for that in the groovy libs, so I'd say we can remove the whole list.

Reply:
ah ok awesome